arch/arm64/boot/dts/qcom/sdm625-xiaomi-daisy.dts

```diff
@@ -249,4 +293,68 @@
         drive-strenght = <2>;
         bias-pull-down;
     };
-};
\ No newline at end of file
+
+    ts_int_default: ts_int_default {
```

Comment:
pinctrl subnode names should end with `-pins` because of pattern chosen in [bindings](https://github.com/torvalds/linux/blob/f5e6c330254ae691f6d7befe61c786eb5056007e/Documentation/devicetree/bindings/pinctrl/qcom%2Cmsm8953-pinctrl.yaml#L47).
Also please convert underscores to hyphens for consistency.

Reply:
Do i have to change the underscores on the tags also?